<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose_Scene</class>
 <widget class="QWidget" name="Choose_Scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关卡选择</string>
  </property>
  <widget class="QPushButton" name="b1_exit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1_nameOfLevel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l2_flower">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/flower_1.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l3_score">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: normal normal 20px &quot;微软雅黑&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
